src/Symfony/Component/Workflow/Dumper/PlantUmlDumper.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Dumper;
+
+use InvalidArgumentException;
+use Symfony\Component\Workflow\Definition;
+use Symfony\Component\Workflow\Marking;
+
+/**
+ * PlantUmlDumper dumps a workflow as a PlantUML file.
+ *
+ * You can convert the generated dot file with the plantuml.jar utility (http://plantuml.com/):
```

Comment:
I suppose it does not generate a dot file.

Reply:
that is correct, I have pushed the correction. thx!